testing/run-scripts/image_make.sh

```diff
@@ -51,7 +55,17 @@ TMP_DIR=`mktemp -d`
 
 cp -R ${BASH_SOURCE%/*}/../integration/test $TMP_DIR/test
 
-cat <<EOF > $TMP_DIR/Dockerfile
+if [ "$DOCKER" = true ]
+then
+  cat <<EOF > $TMP_DIR/Dockerfile
+FROM soycode/uproxy-zork
+
+RUN apt-get -qq update
+RUN apt-get -qq upgrade
```

Comment:
I'm going to flag this myself as "I'm not sure it's needed but decided to add it" - it does seem good to ensure that images are updated, but unattended-upgrades should already do that. Still it'd be nice to ensure that there's a check right at image creation rather than waiting for the cron. It looks like it may be possible to manually invoke unattended-upgrades as a one-time thing, so let me know if you think that's preferable.